framework/src/modules/chain/dpos/delegates_info.js

```diff
@@ -340,4 +347,4 @@ class DelegatesInfo {
 	}
 }
 
-module.exports = { DelegatesInfo };
+module.exports = { DelegatesInfo, EVENT_ROUND_CHANGED };
```

Comment:
Would it make sense to export like this to clearly distinguish between both parameters?
```
module.exports = {
    DelegatesInfo,
    events: {
        EVENT_ROUND_CHANGED,
    }
}
```

Reply:
Allows for more events in the future, otherwise, we have to refactor the interface everywhere if we added it as a third parameter.

Reply:
I prefer to keep it simple @michielmulders. I think it's good the way it is. I wish we could move all events to a single constants file but it's a trivial topic for the moment and can be easily refactored in future imo.

Reply:
Agree, we discussed this as well that it should go in some constants file. To be honest, instead of postponing this I would make more sense to do it now no?

Reply:
For me, it's up to @lsilvs. I'm ok with either way. I think maybe tackling this with a refactoring ticket makes more sense because we don't follow a common pattern when it comes to constant names. Also, there are constants that are changing application behaviour and some of them are just implementation details... So, I don't have a strong opinion on this :)

Reply:
I agree that it would be nice to have all events in one single file but would prefer to do that in a separate ticket. Will leave as it's for now.